src/trace_processor/importers/etw/etw_parser.cc

```diff
@@ -38,10 +41,84 @@ namespace perfetto {
 namespace trace_processor {
 
 namespace {
-
 using protozero::ConstBytes;
 
+static constexpr auto kZeroPageCountBlueprint =
```

Comment:
Given these are not used across functions, can we just put these insnide the file?